src/WebJobs.Script.WebHost/Middleware/SystemTraceMiddleware.cs

```diff
@@ -36,6 +37,11 @@ public async Task Invoke(HttpContext context)
 
             await _next.Invoke(context);
 
+            if (context.RequestAborted.IsCancellationRequested)
```

Comment:
Should we also be catching exceptions incase some later middleware already threw for request aborted?